modules/lookup.js

```diff
@@ -92,27 +97,81 @@ function lookupTerm(message) {
     }); 
 }
 
-function getImageAndSend(message, ID) {
+async function getImageAndSend(message, ID) {
     if (ID < 1)
         throw `Invalid ID given. I can't lookup ${ID}`;
     let tmpdir = tmp.dirSync();
     console.log("Getting screenshot...");
-    let pageres = new Pageres({delay: 0, selector:'article.result', filename:'foo'})
-        .src('https://pf2.easytool.es/index.php?id='+ID, ['1024x768'], {crop: true})
-        .dest(tmpdir.name)
-        .run()
-        .then(function() {
-            console.log('Saving to ' + tmpdir.name+'/'+'foo.png');
-            message.channel.send({files: [{attachment: tmpdir.name+'/'+'foo.png',name:'results.png'}]})
-            .then(msg => {
-                tmp.setGracefulCleanup();
-            })
-            .catch(error => {
-                console.error(error);
-                tmp.setGracefulCleanup();
-            });
-        })
-        .catch(console.error);
+    
+    // temp: test no-sandbox args
+    // args: ['--no-sandbox', '--disable-setuid-sandbox'] - does not work
+    
+    // launchOptions: {args: ['--no-sandbox', '--disable-setuid-sandbox']} - this is what needs to be passed to pageres -> capture-website, 
+    // but pageres filters it out in finalOptions
+    // https://github.com/sindresorhus/pageres/blob/master/source/index.ts
+    // https://www.npmjs.com/package/capture-website
```

Comment:
I don't think this is needed any longer now that migration away from pageres is complete.

Reply:
Removed all the comments and pageres code